scripts/sexbound/lib/sexbound.lua

```diff
@@ -336,23 +338,48 @@ function Sexbound:addActor(actorConfig, store)
     --actor:getApparel():sync()
 
     --actor:initPlugins()
-    
-    if actor:getEntityType() == "player" then self._containsPlayer = true end
-    
+
     self._positions:filterPositions(self._actors)
     
     self._UI:refresh()
-    actor:openUI()
+    
+    if actor:getEntityType() == "player" then 
+        self._containsPlayer = true 
+        -- Open UI if not defeated or is allowed to while defeated.
+        if not actor:getStatus():hasStatus("sexbound_defeated") then
+            actor:openUI()
+            self._playerControl = true
+        else
+            if actor:getStatus():hasStatus("sexbound_defeated_can_use_ui") then
```

Comment:
Naming inconsistency with how it's named on the entity "can_use_sex_ui_defeated".

Reply:
Will update this.